app/app.go

```diff
@@ -463,24 +465,48 @@ func NewGaiaApp(
 	// CanWithdrawInvariant invariant.
 	// NOTE: staking module is required if HistoricalEntries param > 0
 	app.mm.SetOrderBeginBlockers(
-		upgradetypes.ModuleName,
-		capabilitytypes.ModuleName,
-		minttypes.ModuleName,
-		distrtypes.ModuleName,
-		slashingtypes.ModuleName,
-		evidencetypes.ModuleName,
+		crisistypes.ModuleName,
+		govtypes.ModuleName,
 		stakingtypes.ModuleName,
 		liquiditytypes.ModuleName,
+		ibctransfertypes.ModuleName,
 		ibchost.ModuleName,
 		routertypes.ModuleName,
+		capabilitytypes.ModuleName,
+		authtypes.ModuleName,
+		banktypes.ModuleName,
+		distrtypes.ModuleName,
+		slashingtypes.ModuleName,
+		minttypes.ModuleName,
+		genutiltypes.ModuleName,
+		evidencetypes.ModuleName,
+		authz.ModuleName,
+		feegrant.ModuleName,
+		paramstypes.ModuleName,
+		upgradetypes.ModuleName,
+		vestingtypes.ModuleName,
```

Comment:
Hi @robert-zaremba, @alexanderbez , does the order of`moduleNames` in `SetOrderBeginBlockers`, `SetOrderEndBlockers ` and `SetOrderInitGenesis ` functions matter ?

should `auth` comes after `bank` and `staking` in `SetOrderInitGenesis ` function ?
Thank you!

Reply:
is this still an open question?

Reply:
> does the order ofmoduleNames in SetOrderBeginBlockers, SetOrderEndBlockers and SetOrderInitGenesis functions matter ?

Some yes. Unfortunately the easiest way currently to know the order is to follow the comments.

auth and bank have empty begin and end blockers, their order don't matter.